### DISCUSSION TITLE
What React means?

### DISCUSSION THREAD

**Discussion by ra0321** [community/community#37017] [Other Feature Feedback, Questions, & Ideas] [answered]
What React means?

**Comment by jupiterwebdev** [comment, marked as answer]
React is a declarative, efficient, and flexible JavaScript library for building user interfaces. It lets you compose complex UIs from small and isolated pieces of code called “components”. React has a few different kinds of components, but we'll start with React.Component subclasses: class ShoppingList extends React.

**Comment by jupiterwebdev** [comment]
React is a declarative, efficient, and flexible JavaScript library for building user interfaces. It lets you compose complex UIs from small and isolated pieces of code called “components”. React has a few different kinds of components, but we'll start with React.Component subclasses: class ShoppingList extends React.
